lwr:

  # Publish all joint states -----------------------------------
  kuka_joint_state_controller:
    type: lwr_controllers/KUKAJointStateController
    publish_rate: 500  
    root_name: lwr_base_link
    tip_name: lwr_7_link

  joint_controllers:
    type: lwr_controllers/JointControllers
    root_name: lwr_base_link
    tip_name: lwr_7_link
    publish_rate: 500
    max_ee_dt : 20
    max_ee_f : 10
    max_qdot : 60
